l10n_it_reverse_charge/models/account_invoice.py

```diff
@@ -416,13 +417,14 @@ def invoice_validate(self):
 
     def remove_rc_payment(self):
         inv = self
-        if inv.payment_move_line_ids:
-            if len(inv.payment_move_line_ids) > 1:
+        if inv.rc_self_invoice_id.payment_move_line_ids:
+            if len(inv.rc_self_invoice_id.payment_move_line_ids) > 1:
                 raise UserError(
                     _('There are more than one payment line.\n'
                       'In that case account entries cannot be canceled'
                       'automatically. Please proceed manually'))
-            payment_move = inv.payment_move_line_ids[0].move_id
+            payment_move = inv.rc_self_invoice_id.payment_move_line_ids[0].\
```

Comment:
Queste modifiche funzionano anche per https://github.com/OCA/l10n-italy/blob/4ad2fa20dea4e52184148a4f13fa188a90424f9d/l10n_it_reverse_charge/models/account_invoice.py#L477?
Potresti mostrarmi il punto in cui è eventualmente coperto dai test automatici?

Reply:
Per sicurezza l'ho aggiunto: https://github.com/OCA/l10n-italy/compare/b2e849958412a0e475fb103bd94b4647d0a835f1..edcad56999a5902d4d26aad00253ae1d19148bd7